examples/hello_2018/src/main.rs

```diff
@@ -9,3 +10,8 @@ fn hello() -> &'static str {
 fn rocket() -> rocket::Rocket {
     rocket::ignite().mount("/", rocket::routes![hello])
 }
+
+#[rocket::catch(404)]
+fn not_foundr(_req: &'_ rocket::Request<'_>) -> String {
+    "404 Not Found".to_owned()
```

Comment:
There is no reason to allocate here. This example should be fixed to return an `&'static str`.